sdk/tests/conformance2/glsl3/vector-dynamic-indexing.html

```diff
@@ -275,6 +275,21 @@
     my_FragColor = vec4(1.0 - f, f, 0.0, 1.0);
 }
 </script>
+<script id="fshaderIndexUniform" type="x-shader/x-fragment">#version 300 es
```

Comment:
Would it be worth calling this test fShaderIndexUniformWithUniform and adding another one which indexes a global variable with a uniform? The test above indexes a local variable with a uniform, which I could imagine being handled differently.